### PR TITLE
[master < T1104] Fix Kafka e2e test

### PR DESCRIPTION
**Pull request by as51340**
### Update 28.08.2023:
- this is not a proper fix and the failure has been observed also with it hence I will close it. The issue stays open.

Not sure if it is a proper fix but the problem is that at the end of the test we are waiting for stream to close but it never closes. Maybe it just needs more time, maybe there is a logical bug. This fix won't mess up anything that's for sure.

[master < Epic] PR
- [ ] Check, and update documentation if necessary
- [ ] Write E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch and the Epic branch
- [ ] Provide the full content or a guide for the final git message

[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [ ] Write a release note here, including added/changed clauses
- [ ] Tag someone from docs team in the comments

### CODE REVIEW

File: tests/e2e/streams/common.py [hunk 1]
```diff
@@ -9,12 +9,12 @@
 # by the Apache License, Version 2.0, included in the file
 # licenses/APL.txt.
 
-import mgclient
-import pytest
 import time
+from multiprocessing import Manager, Process, Value
 
+import mgclient
+import pytest
```
Comment (antoniofilipovic) [inline tests/e2e/streams/common.py:16]: Can you reorder imports here?

**Review by antoniofilipovic — COMMENTED** [review]
Looks good, just reorder imports and that should be it

**Comment by as51340** [comment]
@antoniofilipovic black and flake do it like this automatically

**Review by antoniofilipovic — APPROVED** [review]

**Comment by as51340** [comment]
@vpavicic no need for release note here

**Pull request closed without merging**